### PR TITLE
HELM: Add control for securityContext for capsule pods

### PR DESCRIPTION
**Pull request by zvlb**
Signed-off-by: Zemtsov Vladimir <vl.zemtsov@gmail.com>

Small feature for control securityContext in helm

### CODE REVIEW

**Comment by prometherion** [comment]
Hey @zvlb, thanks for the PR! I think this is strictly related to #680 by @oliverbaehler: would be great if we could add the defaults detected by Oliver, and let run the `ct` job in this PR that we have that ensures `capsule` can start and bind to the network.

File: charts/capsule/templates/daemonset.yaml [hunk 1]
```diff
@@ -34,6 +34,10 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       {{- end }}
       priorityClassName: {{ .Values.priorityClassName }}
+      {{- with .Values.securityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{-- end }}
```
Comment (prometherion) [inline charts/capsule/templates/daemonset.yaml:40]: It should be `{{- end }}`

**Review by prometherion — CHANGES_REQUESTED** [review]
Please, adhere to the semantic commit messages, more info is available [here](https://www.conventionalcommits.org/en/v1.0.0/).

For Helm changes, I use `chore(helm)` as the subject, and remember to bump up the patch version in the `Chart.yaml` file so we can release a new Chart version.

Also, the Chart documentation must be aligned to the new changes, please, issue `make helm-docs` and commit the result.

**Comment by zvlb** [comment]
Oliver has errors with securityContext, like:
`
securityContext:
          allowPrivilegeEscalation: false
          capabilities:
            drop:
            - ALL
          readOnlyRootFilesystem: true
          runAsGroup: 1002
          runAsNonRoot: true
          runAsUser: 1002
`

Maybe now we will deploy a new chart version with control securityContext
Today or tomorrow I will test best practice for securityContext, and set PR with default values

**Comment by prometherion** [comment]
May I ask you to add the required `securityContext` suggested by Oliver, along with your proposed fix?

**Comment by zvlb** [comment]
I reopen this PR soon

**Pull request closed without merging**